docs/support/troubleshooting.md

```diff
@@ -49,60 +48,98 @@ To resolve this, try either one of the following methods:
 
 !!! warning
 
-    Note that some components in `autoware.universe` don't work without CUDA.
+    Note that some components in Autoware require CUDA.
```

Comment:
Autoware Core shouldn't be dependent on CUDA. So I think it's good to specify Autoware Universe here.
What do you think?

Reply:
I apologise if my understanding is flawed, but I thought that Core was intended to be a stable version of Autoware that had been code reviewed, tested etc, and so if Universe required CUDA for perception then it was reasonable to assume that Core would require CUDA as well? 

I also seem to remember reading that there is a plan to remove the CUDA dependence in the future. Would that apply to Core only or Universe as well?

Reply:
`Core` can be regarded as the refined version of `Universe`, but it includes dependency cleanup in addition to code quality improvements.
Since Autoware is desired to be hardware agnostic, we should avoid CUDA that is locked into NVIDIA.
Regarding `Universe`, I believe CUDA is allowed for more efficiency.

We should update the Autoware concepts documentation.
cc @mitsudome-r @xmfcx

Reply:
Okay, something like this?

```suggestion
    Note that some components in Autoware Universe require CUDA.
```

Reply:
Yes, thank you!